elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -750,24 +756,17 @@ class PfeNavigation extends PFElement {
     this._allRedHatToggle.addEventListener("click", this._toggleAllRedHat);
 
     // General keyboard listener attached to the entire component
+    // @todo/bug: figure out why this event listener only fires once you have tabbed into the menu but not if you have just clicked open menu items with a mouse click on Firefox - functions properly on Chrome
```

Comment:
That might be why I had the menuButton event listener?

Reply:
@wesruv I don't think so since the issue is happening at the desktop breakpoint on firefox when the menu button is not present. But I will make a ticket to investigate further.